ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -0,0 +1,69 @@
+
+let register_smapiv2_server (module S: Storage_interface.Server_impl with type context = unit) sr_ref =
+  let module S = Storage_interface.Server(S) in
+  let rpc = S.process () in
+  let dummy_query_result = Storage_interface.({ driver=""; name=""; description=""; vendor=""; copyright=""; version=""; required_api_version=""; features=[]; configuration=[]; required_cluster_stack=[] }) in
+  Storage_mux.register sr_ref rpc "" dummy_query_result
+
+let override_cbt_functions ?vdi_enable_cbt ?vdi_disable_cbt () =
+  let default a b = match a with
+    | Some a -> a
+    | None -> b
```

Comment:
We probably should move away from this, but until we settle on a different stdlib I think you can use https://github.com/xapi-project/stdext/blob/master/lib/opt.mli#L18 (it's used all over xapi anyway)